unikernel.ml

```diff
@@ -368,32 +446,34 @@ struct
                           in
                           Middleware.http_response reqd ~header_list
                             ~title:"Success"
-                            ~data:
-                              (Yojson.Basic.to_string
-                                 (User_model.user_to_json user))
+                            ~data:(User_model.user_to_json user)
                             `OK
                       | Error (`Msg err) ->
                           Middleware.http_response reqd ~title:"Error"
-                            ~data:(String.escaped err) `Bad_request)
+                            ~data:(`String (String.escaped err))
+                            `Bad_request)
                   | _ ->
                       Middleware.http_response reqd ~title:"Error"
-                        ~data:"A user with this name or email already exist."
+                        ~data:
+                          (`String
+                            "A user with this name or email already exist.")
                         `Bad_request
                 else
                   Middleware.http_response reqd ~title:"Error"
                     ~data:
-                      "CSRF token mismatch error. Please referesh and try \
-                       again."
-                    `Bad_request)
+                      (`String
+                        "CSRF token mismatch error. Please referesh and try \
+                         again.") `Bad_request)
         | _ ->
             Middleware.http_response reqd ~title:"Error"
               ~data:
-                (Fmt.str "Register: Unexpected fields. Got %s"
-                   (Yojson.Basic.to_string (`Assoc json_dict)))
+                (`String
+                  (Fmt.str "Register: Unexpected fields. Got %s"
+                     (Yojson.Basic.to_string (`Assoc json_dict))))
```

Comment:
Here it's encoded as a string because it's going to the UI to be displayed in a toast notification.
In some other cases, particularly that which is only an api response, we want to return the json object as is.